packages/presentation/lib/screen/home_page.dart

```diff
@@ -29,25 +48,14 @@ class _HomePageState extends State<HomePage> {
             ),
             textAlign: TextAlign.center,
           ),
-          Text(answer.toString()),
+          Text(snapshot.data?.isPalindrome.toString() ?? 'null'),
```

Comment:
if u will call Text(snapshot.data?.isPalindrome.toString()) (without handling null case), won't null be printed?